test_path_completion_user_expansion fails when $HOME is /

```
_____________________ test_path_completion_user_expansion ______________________

cmd2_app = <cmd2.Cmd object at 0x7f856ded3828>

    def test_path_completion_user_expansion(cmd2_app):
        # Run path with just a tilde
        text = ''
        if sys.platform.startswith('win'):
            line = '!dir ~\{}'.format(text)
        else:
            line = '!ls ~{}'.format(text)
        endidx = len(line)
        begidx = endidx - len(text)
        completions_tilde = cmd2_app.path_complete(text, line, begidx, endidx)

        # Run path complete on the user's home directory
        user_dir = os.path.expanduser('~')
        if sys.platform.startswith('win'):
            line = '!dir {}'.format(user_dir)
        else:
            line = '!ls {}'.format(user_dir)
        endidx = len(line)
        begidx = endidx - len(text)
        completions_home = cmd2_app.path_complete(text, line, begidx, endidx)

        # Verify that the results are the same in both cases
>       assert completions_tilde == completions_home
E       AssertionError: assert ['/python-cmd2/'] == ['python-cmd2/']
E         At index 0 diff: '/python-cmd2/' != 'python-cmd2/'
E         Use -v to get the full diff

tests/test_completion.py:257: AssertionError
```

--- a/CHANGES.md
+++ b/CHANGES.md
@@ -4,13 +4,14 @@ News
 0.7.5
 -----
 
-*Release date: TBD*
+*Release date: 2017-TBD*
 
 * Bug Fixes
     * `case_insensitive` is no longer a runtime-settable parameter, but it was still listed as such
     * Fixed a recursive loop bug when abbreviated commands are enabled and it could get stuck in the editor forever
         * Added additional command abbreviations to the "exclude from history" list
     * Fixed argparse_example.py and pirate.py examples and transcript_regex.txt transcript
+    * Fixed a bug in a unit test which occurred under unusual circumstances
 * Enhancements
     * Organized all attributes used to configure the ParserManager into a single location
     * Set the default value of `abbrev` to `False` (which controls whether or not abbreviated commands are allowed)

--- a/tests/test_completion.py
+++ b/tests/test_completion.py
@@ -238,7 +238,7 @@ def test_path_completion_user_expansion(cmd2_app):
     if sys.platform.startswith('win'):
         line = '!dir ~\{}'.format(text)
     else:
-        line = '!ls ~{}'.format(text)
+        line = '!ls ~/{}'.format(text)
     endidx = len(line)
     begidx = endidx - len(text)
     completions_tilde = cmd2_app.path_complete(text, line, begidx, endidx)

